Actions/Undo.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if UNITY_EDITOR
+﻿
 namespace UnityEditor.Experimental.EditorVR.Actions
 {
     [ActionMenuItem("Undo", ActionMenuItemAttribute.DefaultActionSectionName, 2)]
```

Comment:
This is kind of out there thinking - we're already doing an undo-patch for EXR. Any way we can just shoehorn undo/redo in through that patch?

Reply:
No, the undo system is currently implemented in the editor itself, with C# API hooks. As Amir mentioned, the C# undo hack week project might work